packages/code-du-travail-frontend/src/outils/indemniteLicenciement/ccn/3043/indemnite.js

```diff
@@ -1,6 +1,6 @@
 import { round } from "../../math";
 /**
- * calcul de l'indemnite de licenciement conventionnelle
+ * calcul de l’indemnite de licenciement conventionnelle
```

Comment:
this looks like a wild search/replace :) Not sure it's a good idea to use typography quote in my comments  :)